src/plugins/email/README.md

```diff
@@ -0,0 +1,68 @@
+- infos = Information about the email plugin is in keys below
+- infos/author = Andreas Kraschitzer <e01226853@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = check
+- infos/recommends =
+- infos/placements = presetstorage
+- infos/status = maintained unittest nodep experimental
+- infos/metadata = check/email
+- infos/description = Validation for email addresses
+
+# Email Address Validation
+
+## Introduction
+
+This plugin validates email addresses using regular expressions.
+
+## Installation
+
+See [installation](/doc/INSTALL.md).
+The package is called `libelektra5-experimental`.
+
+## Usage
+
+```sh
+# Mount `email` plugin to cascading namespace `/tests/email`
+kdb mount config.dump /tests/email dump
+
+
+# Check the validity of the email stored in `/tests/email/adr`
+kdb meta-set /tests/email/adr check/email
+
+# Set a correct email address
+kdb set /tests/email/adr test+email@dev.libelektra.com
+kdb get /tests/email/adr
+#> test+email@dev.libelektra.com
+
+# Try to set incorrect addresses
+kdb set /tests/email/adr invalid.address@com
+# RET: 0
```

Comment:
Why does it return 0? And also check for specific ERROR

Reply:
I did some testing and I don't understand. The plugin seems to not be loaded or applied to the key.
What am I missing?

Reply:
You need to add your plugin when you mount the mountpoint.
Judging from the docs you only mount `dump`, so your plugin is not loaded at all.

Something like this should work:
```
kdb mount config.dump /tests/email dump email
```